runtime/kusama/src/lib.rs

```diff
@@ -1624,11 +1624,30 @@ pub type Executive = frame_executive::Executive<
 	frame_system::ChainContext<Runtime>,
 	Runtime,
 	AllPalletsWithSystem,
-	InitiatePoolConfigs,
+	(
+		RenameBagsListToVoterList,
+		pallet_bags_list::migrations::AddScore<Runtime>,
+		InitiatePoolConfigs,
+	),
 >;
 /// The payload being signed in the transactions.
 pub type SignedPayload = generic::SignedPayload<Call, SignedExtra>;
 
+/// A migration which renames the pallet `BagsList` to `VoterList`
+pub struct RenameBagsListToVoterList;
+impl frame_support::traits::OnRuntimeUpgrade for RenameBagsListToVoterList {
+	#[cfg(feature = "try-runtime")]
+	fn pre_upgrade() -> Result<(), &'static str> {
+		// For other pre-upgrade checks, we need the storage to already be migrated.
+		frame_support::storage::migration::move_pallet(b"BagsList", b"VoterList");
```

Comment:
I don't get why we need this in the pre upgrade check?

Reply:
this is just a hack because we have two migrations around the same pallet. All pre-migrations are ran first, then all migrations, then all post migrations. 

We need to mimic the old migrations happening to make it work. 

Perhaps we can change the logic here to run the pre-post hooks per migration, not all at once.

Reply:
Yeah I was thinking the same thing, but not sure how easily implementable that is given the "tuple" trait stuff.